### PR TITLE
Split xbr db

### PR DESCRIPTION
**Pull request by om26er**
Fixes https://github.com/crossbario/cfxdb/issues/30

### CODE REVIEW

File: cfxdb/xbrmm/schema.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    Data encryption key offers.
+    """
+
+    idx_offer_by_key: IndexOfferByKey
+    """
+    Index of key offers by key ID (rather than offer ID, as the object table
+    is indexed by).
+    """
+
+    transaction: Transactions
```
Comment (oberstet) [inline cfxdb/xbrmm/schema.py:78]: shoudl be `transactions` - yes, this isn't the case currently (existing inconsistency), but we should fix this in this PR as well!

File: cfxdb/xbrmm/schema.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        Factory to create a schema from attaching to a database. The schema tables
+        will be automatically mapped as persistant maps and attached to the
+        database slots.
+
+        :param db: zlmdb.Database
+        :return: object of Schema
+        """
+        schema = Schema(db)
+
+        schema.consents = db.attach_table(Consents)
```
Comment (oberstet) [inline cfxdb/xbrmm/schema.py:94]: sorry, after relooking at my issue description: this is a bug in my desc .. we need consents (read-only) from the market maker, but the on-chain submission is done in xbr network backend, and the on-chain information (consents) are replicated and written on a xbr worker at the worker level. hence `consents` (and `idx_consent_by_member_adr`) should be moved to schema `xbr` (from `xbrmm`).

the other tables are correctly placed in this schema, that is: channels, offers and transactions

File: cfxdb/xbrmm/schema.py (hunk 1 above)
Comment (om26er) [reply cfxdb/xbrmm/schema.py]: done

File: cfxdb/xbrmm/schema.py (hunk 2 above)
Comment (om26er) [reply cfxdb/xbrmm/schema.py:94]: done that

**Comment by om26er** [comment]
> so `ActorType`: yeah, lets move it to common.fbs.

That won't be needed anymore, since Consent now lives in the xbr namespace.

**Pull request merged**